src/main/groovy/com/avioconsulting/mule/linter/model/ConfigurationFile.groovy

```diff
@@ -44,6 +93,21 @@ class ConfigurationFile extends ProjectFile {
         List<GPathResult> components = configXml.depthFirst().findAll {
             it.name() == componentType && it.namespaceURI() == namespace
         }
+        comps.each { comp ->
```

Comment:
Where is comps coming from?  I don't see it in this method.

Reply:
I think this may have override some changes kevin made during the merge.

Reply:
fixed.